I/IGLWrap/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+#=
+BINARYBUILDER_RUNNER=privileged julia build_tarballs.jl --verbose --debug       
+=#
+
+using BinaryBuilder
+using Pkg
+
+version = v"2.3.0" # libIGL version, see below:
+
+sources = [
+	DirectorySource("./iglwrap"),
+	# 2.3.0 stable release:
+	GitSource("https://github.com/libigl/libigl.git",
+		"e60423e28c86b6aa2a3f6eb0112e8fd881f96777")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir build||true; cd build
+cmake \
+  -DCMAKE_INSTALL_PREFIX=$prefix \
+	-DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
+	..
+make && make install
+
+# install_license ${WORKSPACE}/srcdir/LICENSE
```

Comment:
You must set the toolchain file: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds
```suggestion
cd $WORKSPACE/srcdir
mkdir -p build
cd build
cmake \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
    ..
make -j${nproc}
make install

# install_license ${WORKSPACE}/srcdir/LICENSE
```
What happened to the license file?